code/game/machinery/hologram.dm

```diff
@@ -509,7 +511,7 @@ For the other part of the code, check silicon say.dm. Particularly robot talk.*/
 
 /obj/machinery/holopad/proc/move_hologram(mob/living/user, turf/new_turf)
 	if(LAZYLEN(masters) && masters[user])
-		var/obj/effect/overlay/holo_pad_hologram/holo = masters[user]
+		holo = masters[user]
```

Comment:
Storing it up there will cause hard-dels won't it, since the reference will continue after this proc completes. I don't see anywhere this is ever unset